optional_plugins/varianter_yaml_to_mux/tests/test_mux.py

```diff
@@ -533,6 +533,49 @@ def test_false_direct_parent(self):
         self.assertNotEqual(mux.path_parent('/os/linux'), '/')
 
 
+class TestCreateFromYaml(unittest.TestCase):
+
+    def test_normalize_path(self):
+        self.assertEqual(yaml_to_mux._normalize_path(''), None)
+        self.assertEqual(yaml_to_mux._normalize_path('path'), 'path/')
+
+    def test_handle_control_path_include_file_does_not_exist(self):
+        self.assertRaises(ValueError,
```

Comment:
Dont you think using `assertRaises` as a context manager makes code more readable? I think it make more explicit the code being called.

Reply:
It does indeed.